src/components/AssetIcon.tsx

```diff
@@ -119,3 +117,31 @@ export const AssetIcon = memo(({ assetId, showNetworkIcon, src, ...rest }: Asset
 
   return <Avatar src={src} bg={assetIconBg} icon={foxIcon} {...rest} />
 })
+
+type AssetOnLedgerProps = {
+  assetId: AssetId
+} & AvatarProps
+
+export const AssetOnLedger: React.FC<AssetOnLedgerProps> = ({ assetId, size, ...rest }) => {
```

Comment:
Not a blocker by any means but not sure how I feel about this living in this top-level shared component(s) module, would probably make more sense to colocate in Ledger/Accounts Management domain

Reply:
Yeah I hear that - moved to the account management domain in https://github.com/shapeshift/web/pull/6790/commits/761ec0a6a9118087ac99aefe414df0fc007bed05.